crates/wasm/src/lib.rs

```diff
@@ -74,6 +77,20 @@ pub fn get_address_by_index(full_viewing_key: &str, index: u32) -> JsValue {
     return serde_wasm_bindgen::to_value(&address_str).unwrap();
 }
 
+#[wasm_bindgen]
+pub fn get_ephemeral_address(full_viewing_key: &str, index: u32) -> JsValue {
+    utils::set_panic_hook();
+    let fvk = FullViewingKey::from_str(full_viewing_key.as_ref())
+        .context("The provided string is not a valid FullViewingKey")
+        .unwrap();
+
+    let (address, _dtk) = fvk.ephemeral_address(OsRng, index.into());
```

Comment:
How does `OsRng` work in the web context? Does it just work?

Reply:
Yeah, I believe the crate has compile-time features that hook the platform APIs.